### ISSUE TITLE
Alarm widget cannot show alarms of multiple devices? thingsboard 2.3.1

### ISSUE THREAD

**Issue by lingongzi** [thingsboard/thingsboard#1762]
In tb1.4, alert widget will display the gateway device and it can display the alerts for all the devices under the gateway. But in tb2.3, with the same configuration, it can only display one device under the gateway.
So in tb 2.3, how can it display the alerts for multiple devices? I have tried all possible ways but still not work. Thanks for helping.

**Comment by mp-loki** [comment]
Create a relation between your devices and gateway device (if it's not already there) and use relations query when creating an alias

**Comment by Tasosgi** [comment]
Ι observe the some. The Device Gateway_SN7 related with a number of devices via "Created" relation. I use a relation query but only one device displayed at the alarm widget.
At rule chain  action CREATE ALARM I have enable the Propagate

![Capture](https://user-images.githubusercontent.com/31473443/70907057-8750b380-2010-11ea-9d55-23a1200254ff.JPG)
![Capture2](https://user-images.githubusercontent.com/31473443/70907063-89b30d80-2010-11ea-8766-68267bff3748.JPG)

**Comment by thucar** [comment]
Has this been addressed? Is this a configuration issue or a bug? I'm seeing exact same behavior.

**Comment by ecmap** [comment]
Entity group list > Type:Device > Device type ?

**Comment by thucar** [comment]
@ecmap I take it this only applies to paid version and not the Community edition?

**Comment by thucar** [comment]
I can not believe it I actually managed to get it to work. It's incredibly non intuitive and feels like a workaround and not a solution.

I have this setup: House(asset)-Room(asset)--Gateway(device)--Device(device)

I ended up creating an alias like this: 
Relations query
Root entity type: Asset
Root entity Asset: Room01
Direction: To
Max relation level:1
No filters

If I tried it the other way around and said that root entity is House and used relations direction "from" it would not work.
